paf-mvp-demo-express/src/views/publisher/index.hbs

```diff
@@ -96,6 +96,20 @@
         pbjs.que.push(function() {
             pbjs.addAdUnits(adUnits);
             pbjs.setConfig({
+                consentManagement: {
+                cmpApi: "static",
+                consentData: {
+                    getTCData: {
+                        tcString: "CO-HDlqO-HDlqAKAXCENBDCsAP_AAH_AACiQHKNd_X_fb39j-_59_9t0eY1f9_7_v20zjgeds-8Nyd_X_L8X42M7vF36pq4KuR4Eu3LBIQFlHOHcTUmw6IkVqTPsak2Mr7NKJ7PEinMbe2dYGHtfn9VTuZKYr97s___z__-__v__75f_r-3_3_vp9V---_fA5QAkw1L4CLMSxwJJo0qhRAhCuJDoAQAUUIwtE1hASuCnZXAR-ggYAIDUBGBECDEFGLIIAAAAAkoiAkAPBAIgCIBAACAFSAhAARoAgsAJAwCAAUA0LACKAIQJCDI4KjlMCAiRaKCeSMASi72MMIQyigBoFH4AAAAA.cAAAAAAAAAAA",
```

Comment:
LGFM as a quick fix. To well understand, to have a better implementation - with a fictive CMP including the iAB specs - would looks like something like that:

````
const callback = (tcData, success) => {

  if(success && tcData.eventStatus === 'tcloaded') {
      pbjs.que.push(function() {
          pbjs.setConfig({
          // ....
         tcString: tcData. tcString
    })
  }

}
__tcfapi('addEventListener', 2, callback);
````
https://github.com/InteractiveAdvertisingBureau/GDPR-Transparency-and-Consent-Framework/blob/master/TCFv2/IAB%20Tech%20Lab%20-%20CMP%20API%20v2.md